pkg/geoipupdate/database/http_reader.go

```diff
@@ -47,6 +48,11 @@ func NewHTTPReader(
 	retryFor time.Duration,
 	verbose bool,
 ) Reader {
+	logger := vars.NewDiscardLogger("reader")
```

Comment:
I'm not sure if we want to have prefixes. @oschwald What are your thoughts?

Reply:
I think ideally the current verbose output would be mostly the same.